src/main/kotlin/com/sourcegraph/cody/edit/widget/LensWidgetGroup.kt

```diff
@@ -349,11 +346,13 @@ class LensWidgetGroup(val session: FixupSession, parentComponent: Editor) :
   }
 
   companion object {
-    private const val LEFT_MARGIN = 100f
+
+    //todo: make it follow the identation of the block. for now it is fixed to the left
```

Comment:
Please use a space after `//` and capitalize TODO; e.g.:

> // TODO:  make it follow the indentation of the block

Reply:
Done!